lib/options.R

```diff
@@ -59,7 +60,18 @@ argument_groups <- list(
 # this method should cat an error and quit() when they are not satisfied
 # or cat a warning message
 validators <- list(
-  cor_level=validate_numeric(min=0, max=1, min_warning=list(.6, "correlation below this level makes no sense"))
+  cor_level=validate_numeric(min=0, max=1, min_warning=list(0.6, "correlation below 0.6 is departing from the spirit of this competition - to group things that likely contain redundant information")),
+  k_splits=validate_numeric(min=2, max_warning=list(6, "these are a lot of splits...using this many splits with small data is probably unwise")),
+  prevalence=validate_numeric(min=0, max=1),
+  abundance=validate_numeric(min=0),
+  lowest_level=validate_numeric(min=1, min_warning=list(2, "values below 2 may include an artificial taxonomic root")),
```

Comment:
maybe in the future we could have the tree offset by one so this makes more sense to a user that doesn't know the model